setup.cfg

```diff
@@ -1,25 +1,110 @@
+[metadata]
+description = Metric logger for ML projects.
+name = dvclive
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+license = Apache-2.0
+license_files = LICENSE
+url = https://github.com/iterative/dvclive
+project_urls =
+    Documentation = https://dvc.org/doc/dvclive
+    Source = https://github.com/iterative/dvclive
+platforms=any
+authors = Paweł Redzyński
+author_email = pawel@iterative.ai
+maintainer = Iterative
+maintainer_email = support@dvc.org
+keywords=data-science, metrics, machine-learning, developer-tools, ai
+classifiers =
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Development Status :: 5 - Production/Stable
+
+[options]
+python_requires = >=3.8
+zip_safe = False
+package_dir=
+    =src
+packages = find:
+install_requires=
+    dvc_render[table]>=0.0.8
+
+[options.extras_require]
+tests =
+    pytest==7.1.2
+    pytest-sugar==0.9.5
+    pytest-cov==3.0.0
+    pytest-mock==3.8.2
+    pylint==2.15.0
+    pylint-plugin-utils>=0.6
+    mypy==0.971
+    pandas>=1.3.1
+    funcy>=1.1
+    dvc>=2.0.0
+    %(all)s
+
+dev =
+    %(tests)s
+image =
+    pillow
+sklearn =
+    scikit-learn
+plots =
+    %(sklearn)s
+mmcv =
+    mmcv
+tf =
+    tensorflow
+xgb =
+    xgboost
+lgbm =
+    lightgbm
+huggingface =
+    transformers
+    datasets
+catalyst =
+    catalyst<=21.12
+fastai =
+    fastai
+pytorch_lightning =
+    pytorch_lightning>=1.6
+all =
+    %(image)s
+    %(mmcv)s
+    %(tf)s
+    %(xgb)s
+    %(lgbm)s
+    %(huggingface)s
+    %(catalyst)s
+    %(fastai)s
+    %(pytorch_lightning)s
+    %(plots)s
+
+[options.packages.find]
+exclude =
+    tests
+    tests.*
+where=src
+
 [flake8]
 ignore=
     E203, # Whitespace before ':'
     E266, # Too many leading '#' for block comment
     W503, # Line break occurred before a binary operator
     P1,  # unindexed parameters in the str.format, see:
     # https://pypi.org/project/flake8-string-format/
-max_line_length=79
+per-file-ignores =
+     tests/test_mmcv.py:C408,
+     # Unnecessary dict call
+max_line_length = 79
 max-complexity = 15
-select=B,C,E,F,W,T4,B902,T,P
-show_source=true
-count=true
-
-[isort]
-profile=black
-known_first_party=dvc,tests
-known_third_party=setuptools,tensorflow
-line_length=79
+select = B,C,E,F,W,T4,B902,T,P
+show_source = true
+count = true
 
-[mypy]
-show_column_numbers = True
-show_error_codes = True
-show_error_context = True
-show_traceback = True
-pretty = True
+[codespell]
+ignore-words-list=
```

Comment:
Ended up adding this because `codespell` was complaining:
```console
src/dvclive/data/plot.py:58: fpr ==> for, far, fps
src/dvclive/data/plot.py:68: fpr ==> for, far, fps
src/dvclive/data/plot.py:73: fpr ==> for, far, fps
src/dvclive/data/plot.py:74: fpr ==> for, far, fps
src/dvclive/data/plot.py:111: fpr ==> for, far, fps
src/dvclive/data/plot.py:121: fpr ==> for, far, fps
src/dvclive/data/plot.py:127: fpr ==> for, far, fps
src/dvclive/data/plot.py:128: fpr ==> for, far, fps
```

Reply:
I usually add this to `.pre-commit-config.yaml`, to avoid adding more config to `setup.cfg`. 

https://github.com/iterative/scmrepo/blob/bbcf8f337cf3ac33764c2c7fc227e8e7bd403cea/.pre-commit-config.yaml#L30

I see that there is a PR for `pyproject.toml` support: https://github.com/codespell-project/codespell/pull/2290.

It's good for now.